content/intro-to-storybook/vue/es/using-addons.md

```diff
@@ -0,0 +1,213 @@
+---
+title: 'Complementos'
+tocTitle: 'Complementos'
+description: 'Aprende a integrar y usar complementos usando un ejemplo popular'
+---
+
+Storybook cuenta con un sistema robusto de [complementos](https://storybook.js.org/addons/introduction/) con el que puede mejorar la experiencia del desarrollador para
+todos en tu equipo. Si ha seguido este tutorial linealmente, hasta ahora hemos hecho referencia a varios complementos, y ya habrá implementado uno en el [Testing](/vue/es/test/).
+
+<div class = "aside">
+<strong> ¿Busca una lista de posibles complementos? </strong>
+<br/>
+😍 Puede ver la lista de complementos de la comunidad con respaldo oficial y con un fuerte respaldo <a href="https://storybook.js.org/addons/addon-gallery/"> aquí </a>.
+</div>
+
+Podríamos escribir para siempre sobre la configuración y el uso de complementos para todos sus casos de uso particulares. Por ahora, trabajemos para integrar uno de los complementos más populares dentro del ecosistema de Storybook: [knobs](https://github.com/storybooks/storybook/tree/master/addons/knobs).
+
+## Configurando Knobs
+
+Knobs ess un recurso increíble para que los diseñadores y desarrolladores experimenten y jueguen con componentes en un entorno controlado sin necesidad de codificar! Básicamente, proporciona campos definidos dinámicamente con los que un usuario manipula los props que se pasan a los componentes de sus historias. Esto es lo que vamos a implementar ...
+
+<video autoPlay muted playsInline loop>
+  <source
+    src="/intro-to-storybook/addon-knobs-demo.mp4"
+    type="video/mp4"
+  />
+</video>
+
+### Instalación
+
+Primero, necesitaremos instalar todas las dependencias necesarias.
+
+```bash
+yarn add @storybook/addon-knobs
+```
+
+Registra Knobs en tu archivo `.storybook/addons.js`.
+
+```javascript
+// .storybook/addons.js
+
+import '@storybook/addon-actions/register';
+import '@storybook/addon-knobs/register';
+import '@storybook/addon-links/register';
+```
+
+<div class = "aside">
+<strong> 📝 ¡El orden de registro de complementos es importante! </strong>
+<br/>
+El orden en que enumere estos complementos determinará el orden en que aparecerán como pestañas en su panel de complementos (para aquellos que aparecen allí).
+</div>
+
+¡Eso es! Es hora de usarlo en una historia.
+
+### Uso
+
+Usemos el tipo de knob de objeto en el componente `Tarea`.
```

Comment:
Change "Tarea" to "Task", if you don't mind, so that coesion between code and documentation will align itself.